client/src/components/Options.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { EvenBiggerContainer, ColoredText, OptContainer, OptionsBox, OptionsIcon, OptPictureBox } from "./styles/styles";
+
+const Options = ({gameId, options}) => {
+    console.log('options', options)
+    return(
+        <div className="options" >
+            <EvenBiggerContainer>
+            {options[0] && <OptContainer>
+                <OptPictureBox>
+                    <OptionsIcon src="./images/ico_singlePlayer.png"/>
+                </OptPictureBox>
+                <OptionsBox>
+                    <ColoredText>Single-Player</ColoredText>
+                </OptionsBox>
+            </OptContainer>}
+            {options[1] && <OptContainer>
```

Comment:
Not a big deal on these but the Air BnB style guide prefers the && operator to be on a new line when doing conditional rendering like this.
<img width="473" alt="Screen Shot 2021-02-10 at 5 14 26 PM" src="https://user-images.githubusercontent.com/66852739/107579658-74f5e980-6bc3-11eb-9c68-f8fbfe19b90f.png">